dactyl-config.yml

```diff
@@ -583,29 +584,261 @@ pages:
 
     -   name: Key Generation Methods #TODO: possible md landing content?
         html: key-generation-methods.html
+        blurb: Use these methods to generate and manage keys.
         funnel: Docs
         doc_type: References
         supercategory: rippled API
-        category: Public rippled Methods
+        category: Admin rippled Methods
```

Comment:
Sorry about the copypaste-o's here and on line 603. Glad you caught and fixed them!